assets/src/routes/routes.js

```diff
@@ -16,6 +16,14 @@ const routes = (courseId, activeViews) => {
       icon: Grade,
       description: 'See what assignments have the greatest impact on your grade.',
       image: '/static/images/assignments_planning_icon.png',
+      viewCode: 'apv1'
+    },
+    {
+      path: `/courses/${courseId}/assignmentsv2`,
+      title: 'Assignment Planning V2',
+      icon: Grade,
+      description: 'See what assignments have the greatest impact on your grade.',
+      image: '/static/images/assignments_planning_icon.png',
```

Comment:
I guess we'll need to create a new screenshot for this view?

Reply:
Yes